lib/FIR/statements.h

```diff
@@ -592,6 +580,14 @@ class Statement : public SumTypeMixin<std::variant<ReturnStmt,  //
     parent->insertBefore(this);
   }
   std::string dump() const;
+
+  static std::size_t offsetof_impl() {
+    return reinterpret_cast<std::size_t>(&static_cast<Statement *>(nullptr)->u);
```

Comment:
I think this is undefined behavior, dereferencing a nullptr. Why do you need this?